addons/escoria-core/game/core-scripts/resources/esc_animationresource.gd

```diff
@@ -0,0 +1,29 @@
+# Resource containing all defined animations and angles for 
+# characters movement.
+ 
+extends Resource
+class_name ESCAnimationResource
+
+
+# Array containing the different angles available for animations.
+# Each angle is defined by an array [start_angle, angle_size].
+# start_angle must be between 0 and 360.
+# Angle 0 and 360 are the same and correspond to UP/NORTH
+# 90 is RIGHT/EAST, 180 is DOWN/SOUTH, etc
+export(Array) var dir_angles = []
```

Comment:
You can use _get_property_list and a little hack to create an array of a specific resource in the inspector like this: https://github.com/deep-entertainment/egoventure/blob/8fe70a5e24cfb92c9efc1916cb7930fe96252a48/addons/egoventure/nodes/hotspots/trigger_hotspot.gd#L146

I'd suggest creating a special Resource for the items of the array and use the hack to make it easier for the game dev.

Reply:
Done